docs/examples.md

```diff
@@ -0,0 +1,363 @@
+Both the `add-labels!` and `remove-labels!` functions takes a lookup representation as second argument.
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+#### Add labels
+
+~~~clojure
+(client/add-labels! conn {:id 45} [:person :salesman])
+~~~
+
+#### Remove labels
+
+~~~clojure
+(client/remove-labels! conn {:labels [:person] :props {:first-name "Neo"}} [:person :salesman])
+~~~
+
+### Update Node or Relationship (properties)
+
+To change the properties of a node or relationship we have added two convenience functions.
+Both the `update-properties!` and `replace-properties!` functions takes a lookup representation as second argument.
+
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+#### Update properties
+
+This function will update an existing properties map based on the following rules.
+
+⋅⋅* Keys existing only in the given property map is added to the object
+..* Keys existing only in the property map on the found object is kept as is
+..* Keys existing in both property maps are updated with values from the given property map
+
+~~~clojure
+(client/update-props! conn {:labels [:person] :first-name "Thomas"} {:first-name "Neo"})
+~~~
+
+#### Replace properties
+
+This function will replace an existing property map with the given one.
+
+~~~clojure
+(client/replace-props! conn {:labels [:person] :first-name "Thomas"} {:last-name-only "Anderson"})
+~~~
+
+### Delete Node or Relationship
+
+The `delete!` function takes a lookup representation as second argument and deletes all matches.
+
+Notice: You need to ensure you have deleted all relatiohsips to a node before you can delete the node.
+
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+~~~clojure
+(client/delete! conn {:labels [:person] :first-name "Neo"})
+~~~
+
+
+## Graph CRUD
+
+This section shows how to do basic CRUD operations on a whole graph through Neo4clj convenience functions.
+
+### Create Graph
+
+To make it easier to create nodes and relationships, we have made a function to do it in one single call.
+
+The `lookups` key allows you to lookup and refer existing nodes in the Neo4j database. The entries are lookup representations.
+The `nodes` key dictates what to nodes create. The entries are node representations.
+The `relatioships` key dictates what to create. The entries are relationship representations and it's possible to use lookups ref-id in the `to` and `from` keys.
+The `returns` key dictates which entities to return as the result of the call. This is a vector of ref-id from collections in the other three keys.
+
+To learn more about the Clojure representation of a create graph structure and it's individual parts please see our [representations](representations.md) page
```

Comment:
I think I would delete this line and just embed the link to `representations.md` in the above text with direct links:

    The `lookups` key specifies a list (vector) of [lookup representations](representations.md#lookup) referring existing nodes in the database.

Reply:
Good idea I have updated both graph examples